gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java

```diff
@@ -107,10 +110,36 @@ public void setUp() {
     @Test
     public void handleEhrExtractFromQueue() throws JSONException {
         // process starts with consuming a message from MHS queue
-        sendInboundMessageToQueue("/xml/RCMR_IN030000UK06/payload_part.xml");
+        sendInboundMessageToQueue("/xml/RCMR_IN030000UK06/payload_part.xml", EBXML_PART_PATH);
+
+        // wait until EHR extract is translated to bundle resource and saved to the DB
+        waitAtMost(Duration.ofSeconds(WAITING_TIME)).until(this::isEhrMigrationCompleted);
```

Comment:
What was the thinking behind specifying the Duration here? I'd say that if the integration test doesn't complete within 10 seconds that's very slow, and should be a failure. 30 seconds is unbelievable slow.

Reply:
I think we can change it back to the original 10 sec, it should be fine.